src/it/projects/profile-with-deps-inherit-parent-depMgmt-flatten-dep-all-oss/bom/pom.xml

```diff
@@ -0,0 +1,58 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.codehaus.mojo.flatten.its</groupId>
+    <artifactId>profile-with-deps-inherit-parent-depMgmt-flatten-dep-all-oss-bom</artifactId>
+    <version>0.1-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <properties>
+        <javax.annotations.version>1.3.2</javax.annotations.version>
+    </properties>
+
+    <developers>
+        <developer>
+            <id>hohwille</id>
+            <name>J&#246;rg Hohwiller</name>
+        </developer>
+    </developers>
+
+    <organization>
+        <name>Codehaus</name>
+    </organization>
+
+    <scm>
+        <url>http://svn.codehaus.org/mojo</url>
+    </scm>
+
+    <issueManagement>
+        <system>jira</system>
+        <url>http://jira.codehaus.org/browse/MOJO</url>
+    </issueManagement>
+
+    <distributionManagement>
+        <downloadUrl>http://mojo.codehaus.org/downloads</downloadUrl>
+    </distributionManagement>
+
+    <pluginRepositories>
+        <pluginRepository>
+            <id>none</id>
+            <url>@repository.proxy.url@</url>
+        </pluginRepository>
+    </pluginRepositories>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>javax.annotation</groupId>
+                <artifactId>javax.annotation-api</artifactId>
+                <version>${javax.annotations.version}</version>
```

Comment:
we shouldn't have `<version>` here since this is what we are testing for

Reply:
Already update